spec.html

```diff
@@ -5220,6 +5220,7 @@ <h1>
       </dl>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
+        1. If _number_ is either *+∞*<sub>𝔽</sub> or *-∞*<sub>𝔽</sub>, return *255*<sub>𝔽</sub>.
```

Comment:
shouldn't -∞ clamp to 0?

Reply:
I was following the suggestion from @michaelficarra , but it also makes sense to me that "-∞ would be <= 0 and so it'd return 0."

Reply:
Yeah this was my bad, we should return 0 for -Infinity.

Reply:
```suggestion
        1. If _number_ is *+∞*<sub>𝔽</sub>, return *255*<sub>𝔽</sub>.
        1. If _number_ is *-∞*<sub>𝔽</sub>, return *0*<sub>𝔽</sub>.
```